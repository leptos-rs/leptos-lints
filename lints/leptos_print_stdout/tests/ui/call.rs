fn main() {
    leptos::logging::log!("foo");

    // This should not trigger the lint
    #[allow(unknown_lints)]
    #[expect(leptos_print_stdout)]
    {
        leptos::logging::log!("this should not trigger the lint");
    }

    ::leptos::logging::log!("bar");
}
