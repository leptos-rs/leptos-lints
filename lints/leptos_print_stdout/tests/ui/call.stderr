warning: use of `leptos::logging::log!`
  --> $DIR/call.rs:2:5
   |
LL |     leptos::logging::log!("foo");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_print_stdout#readme
   = note: `#[warn(leptos_print_stdout)]` on by default

warning: use of `leptos::logging::log!`
  --> $DIR/call.rs:11:5
   |
LL |     ::leptos::logging::log!("bar");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://github.com/leptos-rs/leptos-lints/tree/main/lints/leptos_print_stdout#readme

warning: 2 warnings emitted

